tests/functional/ovndbcluster_controller_test.go

```diff
@@ -73,6 +74,19 @@ var _ = Describe("OVNDBCluster controller", func() {
 			Entry("config-data CM", "config-data"),
 			Entry("scripts CM", "scripts"),
 		)
+		DescribeTable("should not create the dnsdata CR",
+			func(DnsName string) {
+				Eventually(func() *infranetworkv1.DNSData {
+					dnsm := types.NamespacedName{
+						Namespace: OVNDBClusterName.Namespace,
+						Name:      fmt.Sprintf("dns-%s", OVNDBClusterName.Name),
+					}
+					return GetDNSData(dnsm)
+				}, timeout, interval).Should(BeNil())
+			},
+			Entry("config-data CM", "config-data"),
```

Comment:
config maps references?

Reply:
At that time I was starting with functional testing, it should be correct now